client/order/index.js

```diff
@@ -135,221 +135,209 @@ jQuery( function ( $ ) {
-				// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-				dispute_default: __(
-					// eslint-disable-next-line max-len
-					'This order has been disputed in the amount of %1$s. The customer provided the following reason: %2$s. Please respond to this dispute before %3$s.',
-					'woocommerce-payments'
-				),
-				// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-				inquiry_default: __(
-					// eslint-disable-next-line max-len
-					'The card network involved in this order has opened an inquiry into the transaction with the following reason: %2$s. Please respond to this inquiry before %3$s, just like you would for a formal dispute.',
-					'woocommerce-payments'
-				),
-				// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-				dispute_urgent: __(
-					'Please resolve the dispute on this order for %1$s labeled "%2$s" by %3$s.',
-					'woocommerce-payments'
-				),
-				// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-				inquiry_urgent: __(
-					'Please resolve the inquiry on this order for %1$s labeled "%2$s" by %3$s.',
-					'woocommerce-payments'
-				),
-			};
-			const amountFormatted = formatExplicitCurrency(
+	// If the dispute is due in the past, don't show notice.
+	if ( ! now.isBefore( dueBy ) ) {
+		return null;
+	}
+
+	return (
+		<DisputeNotice
+			chargeId={ chargeId }
+			disputeReason={ dispute.reason }
+			formattedAmount={ formatExplicitCurrency(
 				dispute.amount,
 				dispute.currency
-			);
-
-			let buttonLabel = __( 'Respond now', 'woocommerce-payments' );
-			let suffix = '';
-
-			let titleText = isInquiry( dispute )
-				? titleStrings.inquiry_default
-				: titleStrings.dispute_default;
-
-			// If the dispute is due within 7 days, use different wording.
-			if ( countdownDays < 7 ) {
-				titleText = isInquiry( dispute )
-					? titleStrings.inquiry_urgent
-					: titleStrings.dispute_urgent;
-
-				suffix = sprintf(
-					// Translators: %s is the number of days left to respond to the dispute.
-					_n(
-						'(%s day left)',
-						'(%s days left)',
-						countdownDays,
-						'woocommerce-payments'
-					),
-					countdownDays
-				);
-			}
+			) }
+			isPreDisputeInquiry={ isInquiry( dispute ) }
+			dueBy={ dueBy }
+			countdownDays={ Math.floor( dueBy.diff( now, 'days', true ) ) }
+			disputeDetailsUrl={ disputeDetailsUrl }
+		/>
+	);
+};
 
-			const title = sprintf(
-				titleText,
-				amountFormatted,
-				reasons[ dispute.reason ].display,
-				dateI18n( 'M j, Y', dueBy.local().toISOString() )
-			);
+const DisputeNotice = ( {
+	disputeReason,
+	formattedAmount,
+	isPreDisputeInquiry,
+	dueBy,
+	countdownDays,
+	disputeDetailsUrl,
+} ) => {
+	useEffect( () => {
+		wcpayTracks.recordEvent( 'wcpay_order_dispute_notice_view', {
+			is_inquiry: isPreDisputeInquiry,
```

Comment:
What do you think about providing `dispute_status` rather than `is_inquiry`, in order to capture `under_review`.

I see that this PR has removed the notice for `under_review`, so this may not be required, although it's not clear.

Reply:
I suspect I will need to do this to fix the "dispute under review" scenario – thanks for the tip 🙌🏻

Reply:
Actually, can render that simple case a whole lot earlier.